p4-16/spec/P4-16-spec.mdk

```diff
@@ -4263,6 +4263,24 @@ only if they have the same element type and the same length.  Two
 stacks are equal if and only if all their corresponding elements are
 equal.  Note that the `nextIndex` value is not used in the equality comparison.
 
+### Header stack expressions { #sec-hs-init }
+
+An expression of the following form evaluates to a header stack: '('
+`typeRef` ')' "{" expressionList "}", where typeRef is a header stack
+type name.  Each expression in the list must evaluate to a header of
+the appropriate type.  The number of headers in the list must equal
+the stack size.
+
```

Comment:
The example below greatly clarifies the statement above. Here are some editorial thoughts:
1. We state that `typeRef` is " header stack type *name*`, which might be misconstrued that we do not allow complex typeRefs in here. Perhaps it makes sense to remove the word "name"?
2. In the next sentence it is unclear what "the appropriate type" is. Do we have a term for the "base type" of the header stack (i.e. the type of the individual element"? 
  * Note also that in the example below we require explicit cast for the header stack expression and the invalid header, but not for the headers (of that "appropriate type"). Maybe it would make sense to add that we do support implicit casting in this context?
  * Hopefully, `...` is on the way and will eliminate the need for that explicit invalid header.
3. I would suggest replacing "must equal" with "must be equal to"